### PR TITLE
adapt to  julia-vscode/SymbolServer.jl#194

### PR DESCRIPTION
**Pull request by simeonschaub**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -3,20 +3,20 @@ uuid = "b3cc710f-9c33-5bdb-a03d-a94903873e97"
...
 
 [deps]
-Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
 CSTParser = "00ebfdb7-1f24-5e51-bd34-a7502290713f"
+Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
 SymbolServer = "cf896787-08d5-524d-9de7-132aaa0cb996"
 
+[compat]
+CSTParser = "2.3.1"
+SymbolServer = "5.1.1-DEV"
```
Comment (simeonschaub) [inline Project.toml:12]: This file code jumbled a bit when I updated the environment, but this line should be the only real change

**Comment by simeonschaub** [comment]
CI complains because no compatible version of SymbolServer has been tagged yet

File: src/StaticLint.jl [hunk 2]
```diff
@@ -18,7 +18,16 @@ const Symbol = SymbolServer.stdlibs[:Core][:Symbol]
 const Int = SymbolServer.stdlibs[:Core][:Int]
 const Float64 = SymbolServer.stdlibs[:Core][:Float64]
 const Vararg = SymbolServer.FakeTypeName(Core.Vararg)
-isva(x) = (x isa SymbolServer.FakeTypeName && x.name.name == :Vararg && x.name.parent isa SymbolServer.VarRef && x.name.parent.name == :Core) || (x isa SymbolServer.FakeUnionAll && isva(x.body))
+
+isva(x::SymbolServer.FakeUnionAll) = isva(x.body)
+if Vararg isa Type
```
Comment (pfitzseb) [inline src/StaticLint.jl:23]:
```suggestion
@static if Vararg isa Type
```
maybe?

Comment (simeonschaub) [reply src/StaticLint.jl:23]: I had that at first, but it's in a baremodule, so I didn't bother with trying to explicitly import it.

Comment (simeonschaub) [reply src/StaticLint.jl]: Ok, was easy enough, so I changed it

File: .github/workflows/jlpkgbutler-ci-master-workflow.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ['1.0', '1.1', '1.2', '1.3', '1.4', '1.5']
+        julia-version: ['1.0', '1.1', '1.2', '1.3', '1.4', '1.5', 'nightly']
```
Comment (davidanthoff) [inline .github/workflows/jlpkgbutler-ci-master-workflow.yml:13]: We can leave for now, but this will be undone by package butler down the road.

**Review by davidanthoff — COMMENTED** [review]
Should I tag a StaticLint.jl patch release next, right?

**Comment by davidanthoff** [comment]
@simeonschaub I think now we have a real bug, right? SymbolServer.jl is tagged, but there seems something wrong still?

**Comment by simeonschaub** [comment]
Ah, yes, forgot to test this for older versions locally again. Hopefully should be fixed now.

**Review by davidanthoff — APPROVED** [review]
Fantastic, thanks!

**Pull request merged**